### DISCUSSION TITLE
Allow scroll up and down action on tabs to be disabled

### DISCUSSION THREAD

**Discussion by shalva97** [minbrowser/min#1643] [Ideas]
This is that one annoying things. its easy to accidently scroll on tabs and close them or open task view.

it doesnot seem hard to add a setting for this... and I would be happy...

or this functionality could be changed, when scrolling up or down and there is a scrollbar below tabs, it would scroll tabs horizontally

**Comment by Flashwalker** [comment]
Yeah, it should be optional.
A lot of people scrolls through tabs with mouse

**Comment by Tedd-Codes** [comment]
Im looking into it

**Comment by Tedd-Codes** [comment]
@PalmerAL I found the code for the scrolling and i found the code for the check boxes in settings but i dont know how to connect them, im trying to make this feature but i dont know how to Maker a checkbox change if the school code is toggled. Could you help? The settings code is in pages/settings/index.html and the scroll code is in js/navbar/tabbar.js.
